ui/pages/confirmations/components/confirm/info/info.tsx

```diff
@@ -23,6 +23,7 @@ const Info = () => {
         return TypedSignInfo;
       },
       [TransactionType.contractInteraction]: () => ContractInteractionInfo,
+      [TransactionType.deployContract]: () => ContractInteractionInfo,
```

Comment:
Does this suggest we should rename `ContractInteractionInfo` to `BaseTransactionInfo` if it's going to provide the template for all more specific transaction types?